src/stories/components/other.js

```diff
@@ -187,4 +172,5 @@ storiesOf('Other', module)
             `,
             },
         }
-    );
+    )
+    .add('Prompt', () => <Prompt model="1" />);
```

Comment:
Could you add support for setting the `model` and `text` prop with knobs?
Take a look at the Icon components for an example 
<img width="488" alt="Screenshot 2019-03-11 at 16 13 15" src="https://user-images.githubusercontent.com/6961901/54134673-afbfd780-4418-11e9-8568-98054614e8c5.png">